e2e/runJest.ts

```diff
@@ -86,6 +86,7 @@ function spawnJest(
   }
   const env: NodeJS.ProcessEnv = {
     ...process.env,
+    NO_COLOR: '1',
     FORCE_COLOR: '0',
```

Comment:
So, while opening this PR I caught a _potential_ regression in Babel. There is no agreed-upon meaning on what `FORCE_COLOR` does: according to some it should always force color (https://force-color.org/), because `NO_COLOR` always unconditionally _disables_ color and they are opposites. According to others, `FORCE_COLOR=0` should disable colors.

Apparently Babel switched from one to the other, so I added this `NO_COLOR=1` just to make sure that e2e tests aren't full of colors until when we decide what to do.